### PR TITLE
Validate accessor type

### PR DESCRIPTION
**Pull request by zvdm**

### CODE REVIEW

File: const.go [hunk 1]
```diff
@@ -138,6 +142,10 @@ func (a *AccessorType) UnmarshalJSON(data []byte) error {
 			"MAT3":   AccessorMat3,
 			"MAT4":   AccessorMat4,
 		}[tmp]
+		if !ok {
+			return errors.New(fmt.Sprintf("unknown accessor's type: %s", tmp))
```
Comment (qmuntal) [inline const.go:146]:
```suggestion
			return errors.New(fmt.Sprintf("gltf: unknown accessor's type: %s", tmp))
```

File: const_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		expType AccessorType
+		typeStr []byte
+		wantErr bool
+	}{
+		{"base", AccessorVec3, []byte(`"VEC3"`), false},
+		{"incorrect-type", 100, []byte(`"CUSTOM_TYPE"`), true},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			var accType AccessorType = 100
```
Comment (qmuntal) [inline const_test.go:20]: this should be a test parameter

**Review by qmuntal — CHANGES_REQUESTED** [review]

File: const.go (hunk 1 above)
Comment (zvdm) [reply const.go]: Thank you, added

File: const_test.go (hunk 2 above)
Comment (zvdm) [reply const_test.go]: Thank you, added test arguments - I hope I have understood you correctly

File: const.go [hunk 3]
```diff
@@ -138,6 +142,10 @@ func (a *AccessorType) UnmarshalJSON(data []byte) error {
 			"MAT3":   AccessorMat3,
 			"MAT4":   AccessorMat4,
 		}[tmp]
+		if !ok {
+			return errors.New(fmt.Sprintf("gltf: unknown accessor's type: %s", tmp))
```
Comment (qmuntal) [inline const.go:146]: I missed this yesterday: use `fmt.Errorf` instead of `errors.New` + `fmt.Sprintf`

**Review by qmuntal — CHANGES_REQUESTED** [review]

File: const.go (hunk 3 above)
Comment (zvdm) [reply const.go]: Oh, yes, I've forgotten about this too )) Thank you! Changed

**Review by qmuntal — APPROVED** [review]

**Pull request merged**